lib/instrumentation.ex

```diff
@@ -56,18 +59,30 @@ defmodule OpentelemetryAbsinthe.Instrumentation do
 
   def handle_operation_start(_event_name, _measurements, metadata, config) do
     params = metadata |> Map.get(:options, []) |> Keyword.get(:params, %{})
+    document = params["query"]
+    operation_name = params["operationName"]
+    operation_type = get_operation_type(document)
 
     attributes =
       []
       |> put_if(
         config.trace_request_variables,
         {"graphql.request.variables", Jason.encode!(params["variables"])}
       )
-      |> put_if(config.trace_request_query, {"graphql.request.query", params["query"]})
+      |> put_if(config.trace_request_query, {Atom.to_string(Conventions.graphql_document()), document})
+      |> put_if(config.trace_request_name, {Atom.to_string(Conventions.graphql_operation_name()), operation_name})
+      |> put_if(config.trace_request_type, {Atom.to_string(Conventions.graphql_operation_type()), operation_type})
```

Comment:
This looks a bit busy with all the conversions, maybe they could be moved to separate variables, wdyt?

Reply:
Yeah, I just realized that I don't exactly need the `Atom.to_string/1` part, so I'll take that off and see if it looks any better.